app/controllers/insured/family_members_controller.rb

```diff
@@ -298,6 +260,52 @@ def show_resident_dependent
 
   private
 
+  # rubocop:disable Metrics/CyclomaticComplexity
+  def set_type_role_and_family
+    @type = if params[:resident_role_id].present?
+              'resident'
+            elsif params[:qle_id].present?
+              market_kind = QualifyingLifeEventKind.find(params[:qle_id]).market_kind
+              market_kind == 'individual' ? 'consumer' : 'employee'
+            elsif params[:employee_role_id].present? && params[:employee_role_id] != 'None'
+              "employee"
+            else
+              "consumer"
+            end
```

Comment:
I could also see using a case statement instead of if->else here due to there being 4 conditions, I've seen something similar elsewhere in the app.

Curious what your take is on that. Any reason to choose if-> else over a case statement, or in general?

Reply:
we decided to let it go due to the circumstances